jlibra-integration-tests/src/test/java/dev/jlibra/MoneyTransferIT.java

```diff
@@ -0,0 +1,200 @@
+package dev.jlibra;
+
+import dev.jlibra.admissioncontrol.AdmissionControl;
+import dev.jlibra.admissioncontrol.query.ImmutableGetAccountState;
+import dev.jlibra.admissioncontrol.query.ImmutableQuery;
+import dev.jlibra.admissioncontrol.query.UpdateToLatestLedgerResult;
+import dev.jlibra.admissioncontrol.transaction.*;
+import dev.jlibra.mnemonic.*;
+import dev.jlibra.move.Move;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.apache.commons.lang3.RandomUtils;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.security.Security;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static admission_control.AdmissionControlOuterClass.AdmissionControlStatus.Accepted;
+import static dev.jlibra.mnemonic.Mnemonic.WORDS;
+import static java.lang.String.format;
+import static java.time.Instant.now;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.with;
+import static org.awaitility.pollinterval.FibonacciPollInterval.fibonacci;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * 1. Create two key pairs A and B.
+ * 2. Mint X libras for account represented by key pair A.
+ * 3. Transfer amount Y from A to B and verify the transaction.
+ */
+public class MoneyTransferIT {
```

Comment:
Maybe OT, but I'd not call it money transfer for several reasons. CoinTransfer or LibraTransfer would be better imho.

Reply:
Changed to SimpleTransactionIT, how is that?

Reply:
Yes, that's fine.

Reply:
done